.gitignore

```diff
@@ -23,3 +23,6 @@ TODO
 .go-version
 
 /gardener
+
+# gosec
+gosec-report.sarif
```

Comment:
Right now this doesn't work as expected. Running `make sast-report` outputs

```
Exporting report to <omitted>/go/pkg/mod/github.com/gardener/gardener@v1.106.1/gosec-report.sarif
```

Running `make sast-report` on the gardener/gardener repo:
```
Exporting report to <omitted>/go/src/github.com/gardener/gardener/gosec-report.sarif
```

We rather want the same behaviour, `gosec-report.sarif` to be saved in the repo root and not somewhere under `<omitted>/go/pkg/mod`.
Can we adapt the location of the report file to be also configurable from outside?